test/wrapper_array.jl

```diff
@@ -143,6 +155,13 @@ end
         @test nda == [1000 1000; 30 40]
     end
 
+    @testset "no arguments" begin
+        nds = NamedDimsArray([4], (:x,))  # 1d 1el array (vector with one element)
+        nds2 = NamedDimsArray{()}(fill(4)); # 0d 1el array (scalar)
+        @test (nds[] = 2) == 2
+        @test (nds2[] = 2) == 2
```

Comment:
This seems like it is only testing the return value that is returned by `setindex`
we should also be testing what the value of `nds` and `nds2` are after this operation.